I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang-12.txt

```diff
@@ -0,0 +1,5320 @@
+
+
+            WDM-TE-Node-1                        WDM-TE-Node-2
+    +-----------------------------+      +---------------------------+
+    |      protected              |      |         ROADM             |
+    |      remote OT      Splitter|      |    +--------------+       |
+    |   +------------+   +--------+      |AD  |              | Line  |
+    |   |         TTP|   |     ---o----->o----o------\       | LTP 1 |
+    |   |       +----|   |    /LTP|      |LTP |       \------o-------o->
+  --o-->|       | Tx o-->o---o    |      |    |              |       |
+    |   |       +----|   |    \   |      |AD  |              |       |
+  <-o---|       | Rx o   |     ---o----->o----o---\          | Line  |
+    |   |       +----|   |     LTP|      |LTP |    \         | LTP 2 |
+    |   |            |   +--------+      |    |     \        o-------o->
+    |   +------------+            |      |    |      \       |       |
+    |                             |      o----o       \      |       |
+    |                             |      |AD  |        \     | Line  |
+    |                             |      |LTPs|         \    | LTP 3 |
+    |                             |      |    |          \---o-------o->
+    |                             |      o----o              |       |
+    |                             |      |    |              |       |
+    |                             |      |    +--------------+       |
+    +-----------------------------+      +---------------------------+
+
+
+       Figure 15: OT and OTSI protection function are both in an
+     adjacent WDM-TE-node (protected remote OT, transmit direction)
+
+2.10.2.  OMS MCG protection
+
+   [Editor's note: this section is still work in progress]
+
+   OMS MCG protection is a 1+1 protection architecture where a TE-link
+   between two adjacent WDM-TE-nodes is 1+1 protected.  Such a 1+1
+   protected TE-link composed of an underlying primary and secondary TE-
+   link is shown in Figure 16 together with an unprotected TE-link
+   between the two adjacent WDM-TE-Nodes.
+
+   1+1 OMS MCG protection is a local protection scheme, which can be
+   modeled based on TE-link properties already defined in [RFC8795].
+   The 1+1 protected TE-link is associated with the two underlying TE-
+   links forming the 1+1 protection group together with the splitter/
+   combiner functions in the adjacent WDM-TE-Nodes.  The splitter/
+   combiner functions are not modeled as separate entities in the model.
+   Any optical impairments imposed by the splitter and combiner
+   functions are included in the optical impairments of the ROADM
+   functions in the two adjacent WDM-TE-Nodes (connectivity matrix and
+   LLCL, respectively).
+
+
+
+
+
+Beller, et al.          Expires 14 September 2023              [Page 37]
+
+Internet-Draft    Opt. Impairment-Aware Topo YANG Model       March 2023
+
+
+   Alternatively, it is also possible to model the 1+1 OMS MCG
```

Comment:
It seems worthwhile referencing section 2.10.2.2. here

Reply:
Reference to sub-section added.